spring-integration-mqtt/src/test/java/org/springframework/integration/mqtt/Mqttv5BackToBackAutomaticReconnectTests.java

```diff
@@ -73,23 +71,13 @@ public class Mqttv5BackToBackAutomaticReconnectTests implements MosquittoContain
 
 	@Test
 	public void testReconnectionWhenFirstConnectionFails() throws InterruptedException {
-		Message<String> testMessage =
-				MessageBuilder.withPayload("testPayload")
-						.setHeader(MqttHeaders.TOPIC, "siTest")
-						.build();
-
-		assertThatExceptionOfType(MessageHandlingException.class)
-				.isThrownBy(() -> this.mqttOutFlowInput.send(testMessage))
-				.withCauseExactlyInstanceOf(MqttException.class)
-				.withRootCauseExactlyInstanceOf(UnknownHostException.class);
-
-		connectionOptions.setServerURIs(new String[]{ MosquittoContainerTest.mqttUrl() });
```

Comment:
Didn't you remove this as a collateral damage  just because you have removed a `reconnect()` from the channel adapter when client is created internally?